config/eslint.cjs

```diff
@@ -56,7 +56,8 @@ module.exports = {
       'error',
       { argsIgnorePattern: '^_', varsIgnorePattern: '^_' },
     ],
-    '@typescript-eslint/no-redeclare': ['error'],
+    '@typescript-eslint/no-redeclare': 0,
+    'no-ex-assign': 0,
```

Comment:
Ok, this rule [no-ex-assign](https://eslint.org/docs/latest/rules/no-ex-assign) seems out of question (as discussed)